WordPress/wordpress-develop
# Block Template : Ensure only the main query is modified when resolving template for new posts.

## What?

Add check for the main query before modying it when resolving the template for new posts.

## Why?

Fix 404 response when resolving template for new posts/pages resulting of the wrong query being modified.

## Testing Instructions

* Create a new post
* Look at the sidebar panel "Template" and check it display the correct value instead of "404"

Trac ticket: https://core.trac.wordpress.org/ticket/56058
